### PR TITLE
Added Entitas 0.34.0

### PR DESCRIPTION
**Pull request by sschmid**

### CODE REVIEW

File: docsets/Entitas/docset.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                "Entitas-CSharp",
+                "ECS",
+                "Entity",
+                "Component",
+                "System",
+                "Framework"
+              ],
+
+    "specific_versions": [
+        {
```
Comment (Kapeli) [inline docsets/Entitas/docset.json]: You added the docset to `docsets/Entitas/versions/0.34.0/Entitas.tgz`. If you want specific versions support you need to add it to the JSON too.

Comment (sschmid) [inline docsets/Entitas/docset.json]: Ok thanks, pushed changes to README and docset.json

File: docsets/Entitas/docset.json [hunk 2]
```diff
@@ -19,5 +19,9 @@
         {
             "_comment": "This is optional. Only support specific/older versions if it actually makes sense. This list should be ordered, newest/latest versions should be at the top."
         }
```
Comment (Kapeli) [inline docsets/Entitas/docset.json]: You're missing a comma (`,`) here.

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**